### PR TITLE
Add docs on TensorFlowModel class usage + requirements file for serving

### PR DESCRIPTION
**Pull request by winstonaws**
*Issue #, if available:*

*Description of changes:*
Add docs on TensorFlowModel usage (deploying directly from model artifacts).

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/tensorflow/README.rst [hunk 1]
```diff
@@ -647,6 +647,39 @@ When the ``deploy`` call finishes, the created SageMaker Endpoint is ready for p
...
+  from sagemaker.tensorflow import TensorFlowModel
+
+  tf_model = TensorFlowModel(model_data='s3://mybucket/model.tar.gz',
+                             role='MySageMakerRole',
+                             entry_point='entry.py',
+                             name='model_name')
+
+  predictor = tf_model.deploy(initial_instance_count=1, instance_type='ml.c4.xlarge')
+
+You can also optionally specify a pip requirements if you need to install additional packages into the deployed
```
Comment (laurenyu) [inline src/sagemaker/tensorflow/README.rst]: s/requirements/requirements file - might also be helpful to link to something like https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format

File: src/sagemaker/tensorflow/README.rst [hunk 2]
```diff
@@ -647,6 +647,39 @@ When the ``deploy`` call finishes, the created SageMaker Endpoint is ready for p
...
+runtime environment by including it in your source_dir and specifying it in the 'SAGEMAKER_REQUIREMENTS' env variable:
```
Comment (laurenyu) [inline src/sagemaker/tensorflow/README.rst]: nit: I'd make `'SAGEMAKER_REQUIREMENTS'` monospaced

File: src/sagemaker/tensorflow/README.rst [hunk 3]
```diff
@@ -647,6 +647,39 @@ When the ``deploy`` call finishes, the created SageMaker Endpoint is ready for p
...
+
+  from sagemaker.tensorflow import TensorFlowModel
+
+  tf_model = TensorFlowModel(model_data='s3://mybucket/model.tar.gz',
+                             role='MySageMakerRole',
+                             entry_point='entry.py',
+                             source_dir='my_src', # directory which contains entry_point script and requirements file
+                             name='model_name',
+                             env={'SAGEMAKER_REQUIREMENTS': 'requirements.txt'} # path relative to source_dir
+                             )
```
Comment (laurenyu) [inline src/sagemaker/tensorflow/README.rst]: nit: I'd put the closing parenthesis on the previous line to be consistent with the other code snippet

File: src/sagemaker/tensorflow/README.rst (hunk 1 above)
Comment (winstonaws) [reply src/sagemaker/tensorflow/README.rst]: Good idea, done

File: src/sagemaker/tensorflow/README.rst (hunk 2 above)
Comment (winstonaws) [reply src/sagemaker/tensorflow/README.rst]: Done

**Pull request merged**